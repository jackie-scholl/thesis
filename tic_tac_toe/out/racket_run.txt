
some player won: X wee!!
X O X
O X O
X O X

0
0
some player won: NotOver wee!!
X _ _
_ O _
_ _ _

some player won: NotOver wee!!
X X _
_ O _
_ _ _

some player won: NotOver wee!!
X X O
_ O _
_ _ _

some player won: NotOver wee!!
X X O
_ O _
X _ _

some player won: NotOver wee!!
X X O
O O _
X _ _

some player won: NotOver wee!!
X X O
O O X
X _ _

some player won: NotOver wee!!
X X O
O O X
X O _

some player won: Draw wee!!
X X O
O O X
X O X

