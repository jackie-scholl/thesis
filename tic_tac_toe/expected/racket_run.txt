
some player won: X wee!!
X O X
O X O
X O X

0
0
Game is not yet over; current turn X
X _ _
_ O _
_ _ _

Game is not yet over; current turn O
X X _
_ O _
_ _ _

Game is not yet over; current turn X
X X O
_ O _
_ _ _

Game is not yet over; current turn O
X X O
_ O _
X _ _

Game is not yet over; current turn X
X X O
O O _
X _ _

Game is not yet over; current turn O
X X O
O O X
X _ _

Game is not yet over; current turn X
X X O
O O X
X O _

Game ended in a draw
X X O
O O X
X O X

